ckanext/validation/cli.py

```diff
@@ -1,22 +1,94 @@
-import sys
+# encoding: utf-8
 
 import click
 
-from ckanext.validation.model import create_tables, tables_exist
+from ckanext.validation import common
+
+
+def get_commands():
+    return [validation]
 
 
 @click.group()
 def validation():
-    """Harvests remotely mastered metadata."""
+    """Validation management commands.
+    """
     pass
 
 
-@validation.command()
+@validation.command(name='init-db')
```

Comment:
name is redundant here as click with automatically make underscores in function into hyphens. Unless if that is not true for older click versions

Reply:
I'd say its best to not have ambiguity here as people may be searching for the init-db and not find this due to the function being init_db.